### ISSUE TITLE
Incompatible with Feeder RSS reader

### ISSUE THREAD

**Issue by zackdouglas** [jarrodek/ChromeRestClient#892]
Advanced Rest Client interferes with scrolling of unread items in [Feeder](https://chrome.google.com/webstore/detail/rss-feed-reader/pnjaodmkngahhkoihejjehlcdlnohgmp?utm_source=chrome-app-launcher-info-dialog)'s toolbar pop-out menu.

--
Chrome, 55.0.2883.44 beta (64-bit)
Windows, 7 SP 1

**Comment by jarrodek** [comment]
Hi,

Thank you for issue report.
I'm not sure that this mean though. Can you be more specific?

**Comment by zackdouglas** [comment]
Hi @jarrodek, thanks for being responsive and amenable to my report! I think this was a false alarm, though. I just experienced the issue even with ARC disabled, and I read about another Feeder user having my issue seemingly w/o using ARC.

I'm going to close this on account of being able to reproduce the issue w/o ARC being enabled.

--
(If you know of something that was updated in Chrome v53, the Feeder team might welcome the input.)
